aci/utils.go

```diff
@@ -469,3 +469,13 @@ func setModelAttributes(modelAttr interface{}, modelAttrMap map[string]string) e
 	}
 	return nil
 }
+
+func toggleOptions(option string) string {
+	var values = map[string]string{
+		"enabled":  "yes",
+		"disabled": "no",
+		"yes":      "enabled",
+		"no":       "disabled",
+	}
+	return values[option]
```

Comment:
should we handle situation that key is not found in this function?

Reply:
I don't think it is required.
We always check for the value with GetOk before this function is called in the resource

Reply:
I'll handle the error by returning "" (discussed with Lionel)

Reply:
Ok this was not exactly what I meant. My comment was more as do we need to handle this with an error returned? If empty string is enough then the additional code (see below)  is not needed since not found will always return a "".

`
if output, ok := values[option]; ok {
	return output
} else {
	return ""
}
`

Reply:
Is this resolved?

Reply:
yes it is!